src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -1503,6 +1503,11 @@ private void AddAdditionalTraceContext(MapField<string, string> attributes, Scri
                 {
                     attributes[ScriptConstants.LiveLogsSessionAIKey] = sessionid;
                 }
+                string operationName = context.FunctionMetadata.Name;
+                if (!string.IsNullOrEmpty(operationName))
+                {
+                    attributes[ScriptConstants.OperationNameKey] = operationName;
```

Comment:
Do we have tests that we can update to verify your changes? Otherwise we may regress in the future. I'm assuming we have tests for logging.

Reply:
OperationName has been added to the tests